SkillService.Broker/Consumers/GetUserSkillsConsumer.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LT.DigitalOffice.Kernel.BrokerSupport.Broker;
+using LT.DigitalOffice.Models.Broker.Models.Skill;
+using LT.DigitalOffice.Models.Broker.Requests.Skill;
+using LT.DigitalOffice.Models.Broker.Responses.Skill;
+using LT.DigitalOffice.SkillService.Data.Interfaces;
+using LT.DigitalOffice.SkillService.Models.Db;
+using MassTransit;
+
+namespace LT.DigitalOffice.SkillService.Broker.Consumers
+{
+  public class GetUserSkillsConsumer : IConsumer<IGetUserSkillsRequest>
+  {
+    private readonly IUserSkillRepository _repository;
+
+    private async Task<List<UserSkillData>> GetUserSkillsAsync(IGetUserSkillsRequest request)
+    {
+      List<DbUserSkill> userSkills = await _repository.GetUserSkillsAsync(request.UserId);
+
+      return userSkills.Select(
```

Comment:
add check if userSkills is null (if something went wrong)
return userSkills?.Select(...